### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by hidekazuna**
Required option --provider-os was added to generate-yaml.sh in #90.
This patch fixes README.md.

### CODE REVIEW

**Comment by Lion-Wei** [comment]
Thanks, 
/ok-to-test

File: README.md [hunk 1]
```diff
@@ -37,9 +37,10 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
    ```bash
    cd examples/openstack
-   ./generate-yaml.sh
+   ./generate-yaml.sh --provider-os [os name]
    cd ../..
    ```
+   [os name] is the operating system of your provider environment. Supported Operating Systems: ubuntu, centos.
```
Comment (chaosaffe) [inline README.md]:
```suggestion
   [os name] is the operating system of your provider environment. 
   Supported Operating Systems: 
   - `ubuntu` 
   - `centos`
```

File: cmd/clusterctl/examples/openstack/README.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ configuration is provided.
 
 1. Run the generation script.
 ```
-./generate-yaml.sh
+./generate-yaml.sh --provider-os [os name]
 ```
 
+[os name] is the operating system of your provider environment. Supported Operating Systems: ubuntu, centos.
```
Comment (chaosaffe) [inline cmd/clusterctl/examples/openstack/README.md]:
```suggestion
   [os name] is the operating system of your provider environment. 
   Supported Operating Systems: 
   - `ubuntu` 
   - `centos`
```

**Pull request merged**